docs/readmes/orc8r/architecture_security.md

```diff
@@ -69,10 +69,19 @@ the `certifier` service, to an identity.
 
 At the *southbound interface*, only gateway identities are allowed to make requests. Unregistered gateways are blocked.
 
-At the the *northbound interface*, only operator identities are allowed to make requests. Operators must have ACLs collectively covering
-the requested REST resources.
+At the *northbound interface*, only operator identities are allowed to make requests.
 
-For more information, see the following servicer definitions
+Additionally, the northbound interface is protected by a token that is sent in the header with every request.
 
-- [`access.proto`](https://github.com/magma/magma/blob/master/orc8r/cloud/go/services/accessd/protos/access.proto) access control
-- [`certifier.proto`](https://github.com/magma/magma/blob/master/orc8r/cloud/go/services/certifier/protos/certifier.proto) certificate management
+The admin can create additional users and tokens with desired permissions using the REST API.
+
+All users need to do the following to access REST API resources:
```

Comment:
Here and other file: I generally shy away from "all you need to do is" or "just do this", as it can sometimes come off as condescending. Instead, we can default to being matter-of-fact, e.g. "To access REST API resources, perform the following steps"